src/jit/lsra.h

```diff
@@ -73,6 +73,25 @@ struct LsraBlockInfo
     unsigned int         predBBNum;
     bool                 hasCriticalInEdge;
     bool                 hasCriticalOutEdge;
+
+#if TRACK_LSRA_STATS
+    // Per block maintained LSRA statistics.
+
+    // Number of spills of local vars or tree temps in this baisc block.
```

Comment:
fixed.

---
In reply to: [82913375](https://github.com/dotnet/coreclr/pull/7575#discussion_r82913375) [](ancestors = 82913375)